Makefile.am

```diff
@@ -21,6 +21,7 @@ dist_noinst_SCRIPTS =
 noinst_SCRIPTS =
 check_SCRIPTS =
 noinst_DATA =
+ignore_files =
```

Comment:
@ejohnstown are you okay with adding `ignore_files`? Andras doesn't want these files in `make dist`.

Reply:
`ignore_files` isn't a thing to Automake. That's why it happens to work.

I think it is supposed to be in the group noinst_DATA. (I'm never 100% sure which group stuff like this is supposed to be in.) I think because DATA isn't included in the distribution, and noinst means it won't try to install the files anywhere.